LICENSE

```diff
@@ -1,9 +1,7 @@
-Many of the files in this module have been inherited from the Nordic SDK for
-nRF51822. With the exception of the softdevice, they come with a BSD license
+This module contains softdevice which comes with a BSD license
```

Comment:
you've got two sentences here which are both talking about the softdevice license. The softdevice is covered by the Softdevice License Agreement.